crates/egui/src/containers/collapsing_header.rs

```diff
@@ -6,8 +6,10 @@ use epaint::Shape;
 #[derive(Clone, Copy, Debug)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
 pub(crate) struct InnerState {
-    open: bool,   // Expand / collapse
-    hidden: bool, // Show / Hide
+    /// Expand / collapse
+    open: bool,
+    /// Show / Hide (only on windows)
```

Comment:
I think, that sentence "only on windows" is misleading -- you may think, that you mean Windows OS